### PR TITLE
tests: migrate to docker-services-cli + Postgres 12

### PR DESCRIPTION
**Pull request by ppanero**
Requires https://github.com/inveniosoftware/docker-services-cli/pull/10

### CODE REVIEW

**Comment by ntarocco** [comment]
@topless you will need `'SQLAlchemy>=1.2.18',`

**Comment by ppanero** [comment]
@ntarocco @topless I've squashed my part in one commit

File: setup.py [hunk 1]
```diff
@@ -16,20 +16,15 @@
...
-    'coverage>=4.0',
+    'pytest-invenio>=1.4.0',
     'cryptography>=2.1.4',
-    'isort>=4.2.2',
     'mock>=1.3.0',
-    'pydocstyle>=1.0.0',
-    'pytest-cov>=1.8.0',
-    'pytest-pep8>=1.0.6',
-    'pytest>=3.8.0,<5.0.0',
+    'docker-services-cli>=0.2.0',
```
Comment (lnielsen) [inline setup.py:22]: Minor: you don't need this one, it should be installed by default from ``pytest-invenio``.

**Review by lnielsen — APPROVED** [review]
Just one minor thing about the docker-services-cli dependency.

**Pull request merged**